types/types.go

```diff
@@ -9,14 +9,24 @@ import (
 	"github.com/0xPolygon/polygon-edge/helper/keccak"
 )
 
-var ZeroAddress = Address{}
-var ZeroHash = Hash{}
-
 const (
 	HashLength    = 32
 	AddressLength = 20
 )
 
+var (
+	ZeroAddress = Address{}
+	ZeroHash    = Hash{}
+)
+
+var (
+	// EmptyRootHash is the root when there are no transactions
+	EmptyRootHash = StringToHash("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
+
+	// EmptyUncleHash is the root when there are no uncles
+	EmptyUncleHash = StringToHash("0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347")
+)
```

Comment:
Does it make sense to merge these two blocks? (though I'm ok with having it separated as well)

```suggestion
var (
	ZeroAddress = Address{}
	ZeroHash    = Hash{}

	// EmptyRootHash is the root when there are no transactions
	EmptyRootHash = StringToHash("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")

	// EmptyUncleHash is the root when there are no uncles
	EmptyUncleHash = StringToHash("0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347")
)
```

Reply:
Done